src/_micro-api/libraries/signalDecoder/src/signalDecoder.h

```diff
@@ -43,7 +43,14 @@
 #endif
 
 #ifndef DEC
+/* UB
+no PlattformIO compatible
```

Comment:
Hier über den Kommentar stolpere ich nun schon erneut.
Ich werde es mal testen und ob wir hier noch handeln müssen.

Reply:
Ich verstehe es so, dass DEC in PlattformIO nicht definiert (ist/war)